.mailmap

```diff
@@ -0,0 +1,2 @@
+Jörn Zaefferer <joern.zaefferer@gmail.com>
+Richard D. Worth <rdworth@gmail.com>
```

Comment:
No problem. For clarity about the sign, that's what I was referring to:
![screenshot from 2015-11-26 11-02-31](https://cloud.githubusercontent.com/assets/967155/11423644/5b9d612a-942d-11e5-8fd0-304c804e8e7b.png)


Thanks.